packages/java-spring/src/components/response-with-headers.tsx

```diff
@@ -0,0 +1,70 @@
+import { code } from "@alloy-js/core";
+import * as jv from "@alloy-js/java";
+import { springFramework } from "../spring/libraries/index.js";
+
+/**
+ * Used to represent a response that contains headers. Wraps the actual
+ * response object.
+ */
+export function ResponseWithHeaders() {
```

Comment:
Can we just call this `Response`? Headers are optional and can be part of the response.